vendor/bat-native-ads/src/bat/ads/internal/deprecated/client/client_info.h

```diff
@@ -16,6 +16,7 @@
 #include "bat/ads/internal/deprecated/client/preferences/ad_preferences_info.h"
 #include "bat/ads/internal/serving/targeting/models/behavioral/purchase_intent/purchase_intent_aliases.h"
 #include "bat/ads/internal/serving/targeting/models/contextual/text_classification/text_classification_aliases.h"
+#include "bat/ads/internal/serving/targeting/models/contextual/text_embedding/text_embedding_aliases.h"
```

Comment:
Can we remove changes to `deprectaed/client`?

Reply:
Removed in other branch